src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,148 @@
+import { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from '../../common/Button/Button';
+import Input from '../../common/Input/Input';
+import styles from '../CreateCourse/CreateCourse.module.css';
+import AuthorItem from './components/AuthorItem/AuthorItem';
+import formatCreationDate from '../../helpers/formatCreationDate';
+import createGuid from '../../helpers/createGuid';
+
+const CreateCourse = ({ course, setCourses, author, setAuthor }) => {
+  let [inputs, setInputs] = useState({});
+  let [authorInput, setAuthorInput] = useState('');
+  let [formAuthors, setFormAuthors] = useState([]);
+
+  let authorId = createGuid();
+  let courseId = createGuid();
+
+  const addAuthorToForm = (authorName, event) => {
+    event.stopPropagation();
+    if (formAuthors.includes(authorName)) return;
+
+    setFormAuthors([...formAuthors, authorName]);
+  };
+
+  const deleteAuthor = (authorName) => {
+    setFormAuthors(formAuthors.filter((author) => author !== authorName));
+  };
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setInputs((values) => ({ ...values, [name]: value }));
+  };
+
+  const handleAuthorChange = (event) => {
+    setAuthorInput(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const newCourses = [
+      ...course,
+      {
+        id: courseId,
+        creationDate: formatCreationDate(),
+        authors: formAuthors,
+        ...inputs,
+      },
+    ];
+    setCourses(newCourses);
+
+    setInputs({});
+  };
+
+  const createAuthor = () => {
+    const shouldReturn = (element) => {
+      return element.name === authorInput;
+    };
+
+    author.some(shouldReturn);
+
+    if (authorInput)
+      setAuthor(author.push({ id: authorId, name: authorInput }));
+
+    console.log(author);
+
+    setAuthorInput('');
+  };
+
+  const showCourses = () => {
+    document.getElementById('courses').style.display = 'block';
+    document.getElementById('createCoursePortal').style.display = 'none';
+  };
+
+  return (
+    <Form
+      onSubmit={handleSubmit}
+      id='createCoursePortal'
+      style={{ display: 'none' }}
+    >
+      <Form.Group className='mb-3' controlId='formBasicTitle'>
+        <Form.Label>Title:</Form.Label>
+        <Form.Control
+          type='text'
+          name='title'
+          value={inputs.title || ''}
```

Comment:
Well, this row has not too many sense if you have `''` assigned for `inputs.title`. It's better to use `??` for such cases. Please, read the difference between `||` and `??`